api/src/main/java/io/opentelemetry/metrics/LongValueObserver.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.metrics;
+
+import io.opentelemetry.metrics.LongValueObserver.ResultLongValueObserver;
+import java.util.Map;
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * {@code ValueObserver} is the asynchronous instrument corresponding to ValueRecorder, used to
+ * capture values that are treated as individual with Observe(value).
```

Comment:
all the same comments from the DoubleValueObserver apply in here, as well.

Reply:
Done.